### PR TITLE
Build script for fluent-bit with patches

### PR DESCRIPTION
**Pull request by ManirajDeivendran**
Hi,

Delayed response from the community. So publish build script with patches included PR's.
https://github.com/openresty/luajit2/pull/123
https://github.com/openresty/luajit2/pull/124
https://github.com/fluent/fluent-bit/pull/3591

Need to remove, once it's merged into master.

Kindly review.
Thanks

### CODE REVIEW

**Comment by seth-priya** [comment]
@amitsadaphule please review

**Review by amitsadaphule — CHANGES_REQUESTED** [review]

File: f/fluent-bit/patches/configure [hunk 1]
```diff
@@ -0,0 +1,2 @@
+exit 0;
+
```
[the remark this answers is not in the document]
Comment (ManirajDeivendran) [reply f/fluent-bit/patches/configure:2]: luajit2 require file "configure" which is copied into luajit2 folder for build with fluent-bit.

File: f/fluent-bit/patches/libco.patch [hunk 2]
```diff
@@ -0,0 +1,15 @@
+diff --git a/lib/monkey/deps/flb_libco/libco.c b/lib/monkey/deps/flb_libco/libco.c
+index e0101d23..37bbb65d 100644
+--- a/lib/monkey/deps/flb_libco/libco.c
++++ b/lib/monkey/deps/flb_libco/libco.c
+@@ -16,7 +16,9 @@
+     #include "arm.c"
+   #elif defined(__aarch64__)
+     #include "aarch64.c"
+-  #elif defined(_ARCH_PPC)
++  #elif defined(__powerpc64__) && defined(_CALL_ELF) && (_CALL_ELF == 2)
```
[the remark this answers is not in the document]
Comment (ManirajDeivendran) [reply f/fluent-bit/patches/libco.patch]: Yes. You're right. Modified the code accordingly.

**Comment by ManirajDeivendran** [comment]
@amitsadaphule, Updated the review comments. Please provide your comments if any.

**Review by amitsadaphule — APPROVED** [review]
LGTM

**Pull request merged**